spec/requests/api/v0/highlights_controller_spec.rb

```diff
@@ -250,6 +258,26 @@
           expect(response).to have_http_status :not_found
         end
       end
+
+      context 'when there are two highlights' do
```

Comment:
should there be other variations of the tests:   no neighbors, one neighbor prev, one neighbor after, 2 neighbors?

Reply:
There are a bunch of these variations on the model spec.